Generals/Code/GameEngine/Source/GameNetwork/Transport.cpp

```diff
@@ -146,18 +141,35 @@ Bool Transport::init( UnsignedInt ip, UnsignedShort port )
 	}
 	m_statisticsSlot = 0;
 	m_lastSecond = timeGetTime();
+}
 
-	m_port = port;
+Bool Transport::bind(void)
```

Comment:
You can potentially reduce the diff complexity by moving this function above clearBuffers